src/components/Steps/StepsComponent.js

```diff
@@ -24,37 +20,29 @@ type Props = {
   buttonVisible: boolean
 }
 
-const chooseIcon = (props,icon) => {
-  switch(icon){
-    case 'Persons':
-    if(props.currentStep > 0){return <FaceIcon className={props.classes.completed}/>}
-      return <FaceIcon/>;
-    case 'Family':
-      if(props.currentStep > 1){return <PermContactCalendarIcon className={props.classes.completed}/>}
-      return <PermContactCalendarIcon/>;
-    case 'Home':
-      if(props.currentStep > 2){return <HomeIcon className={props.classes.completed}/>}
-      return <HomeIcon/>;
-    case 'Help':
-      if(props.currentStep >= 3){return <HelpOutlineIcon className={props.classes.completed}/>}
-      return <HelpOutlineIcon/>;
-    default: 
-      break;
+const chooseIcon = (props, index) => {
```

Comment:
This can be refactored. Component should be in main function and this class should only select classes.